### PR TITLE
Pin tools (Node.js, yarn cli, and Rust)

### LINKED ISSUE #460: Update rust to most current version and add it in `.tool-versions`
See https://github.com/serlo/documentation/wiki/How-Tos#how-to-update-the-version-of-nodejs--rust--python on how to do it. It was already done in https://github.com/serlo/database-layer/pull/367 however there was a CI bug in the JS part and thus the PR was closed. You can mainly copy the part for updating rust in this PR.

### PR DESCRIPTION
**Pull request by eliflores**
Fixes #460

### CODE REVIEW

**Comment by AndreasHuber** [comment]
For started sqlx upgrade see https://github.com/serlo/database-layer/tree/upgrade-sqlx-to-0.7.0. It needs some refactoring to compile again.

**Comment by AndreasHuber** [comment]
Why would a Rust version upgrade require changes like sqlx? I think it shouldn't. 🤔

**Review by AndreasHuber — CHANGES_REQUESTED** [review]

**Comment by eliflores** [comment]
Hi @AndreasHuber 
Thank you for the review!

> Why would a Rust version upgrade require changes like sqlx? I think it shouldn't. 🤔
After I changed the version of Rust to 1.70.0, I needed to re-install `sqlx-cli` and since I did not specify a version, the version ` v0.7.0` was installed (since it's the latest release), if I specify the version (i.e. `cargo install sqlx-cli --version 0.6.3`) then we don't need to upgrade =)

File: README.md [hunk 1]
```diff
@@ -2,11 +2,17 @@
...
-The database layer provides a Restful API in front of the database of [Serlo](https://serlo.org/).
+The database layer provides a RESTful API in front of the database of [Serlo](https://serlo.org/).
 
 ## Setup
 
-You need [Docker](https://docs.docker.com/engine/installation/) and [Rust](https://www.rust-lang.org) installed on your system. In order to run the pact tests (= contract test suite we use) you need to install [Node.js](https://nodejs.org/) version `16.x` and [yarn](https://yarnpkg.com/).
+You need the following installed on your system:
+
+- [Docker](https://docs.docker.com/engine/installation/)
+- [Rust](https://www.rust-lang.org) version from [.tool-versions](.tool-versions). [^1]
```
[the remark this answers is not in the document]
Comment (eliflores) [reply README.md:12]: Oh, interesting. 
What would be your suggestion on what we could write on this line instead?

**Review by AndreasHuber — APPROVED** [review]

**Pull request merged**